### DISCUSSION TITLE
EQ Readiness Feedback

### DISCUSSION THREAD

**Discussion by tdegeorge** [TeamBasedInquiryLearning/precalculus#282] [General]
Author: @AbbyANoble 
Reviewer: @tdegeorge 

Link: https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EQ.html

**Comment by tdegeorge** [comment]
For 7 - when I click on the link, it starts with multiplying polynomials first before going into factoring. It is all part of the same unit, which I think is good. Do we want to somehow jump to factoring or add in something about multiplying polynomials?

**Reply by AbbyANoble** [reply]
I couldn't figure out how to jump to the factoring part without picking just one of the sections. I'm updating to include multiple links to all the factoring techniques. Let me know what you think!

**Reply by AbbyANoble** [reply]
Nevermind. I think I'll just link the summary one: https://www.khanacademy.org/math/algebra/x2f8bb11595b61c86:quadratics-multiplying-factoring/x2f8bb11595b61c86:factor-quadratics-strategy/a/factoring-quadratics-in-any-form

It reviews the techniques and has practice. And has links at the top to the previous ones if needed. This should be review, so hopefully this overview is enough!

**Reply by tdegeorge** [reply]
I think that is probably good enough!

**Comment by tdegeorge** [comment]
For 8 - I tried to watch the video, but it stopped me and asked me to create an account. I googled study.com to see if there is a way to view the video without it and I got this:

"The free student accounts only allow students to see lessons that are assigned to them, while the licensed accounts give students the opportunity to explore everything on our site at any time."

Are you able to see the whole thing? I don't know if we want students to use something that they have to create an account for? At least with IXL, they can do some first before having to create an account....

**Reply by AbbyANoble** [reply]
Oops. I didn't realize it did that! I had a hard time finding something that just focused on d=rt and didn't go into the more complicated problems that are part of our lessons. Maybe instead of this being in readiness (which is already very long for this chapter!!) I could add in a simple question after [Remark 1.2.2](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EQ2.html#EQ2-3-3)? Maybe split up the remark and have an example after it states d=rt, then put the second paragraph as its own remark right before the first activity?

**Reply by tdegeorge** [reply]
Do we like any of these videos?
https://www.youtube.com/watch?v=V_L0we4CjrU
https://www.youtube.com/watch?v=Zl6zUBCgLbU
https://www.youtube.com/watch?v=lMO1L_CvH4Y

Or you could add more to the remark and add an activity. :)

**Reply by AbbyANoble** [reply]
Thanks! Added those in to readiness!

**Comment by tdegeorge** [comment]
For 10  - do we want to be more specific and say "simplify square roots..."? The Khan academy only does square roots. 

Actually, now I am not sure. I see 11 is with imaginary and I think I like just roots....

**Reply by AbbyANoble** [reply]
Square roots does make more sense I think. The motivation was so they could use quadratic formula which is always a square root.
